tests/ordering.test.js

```diff
@@ -153,4 +153,39 @@ describe('Ordering', () => {
       storedValue,
     );
   });
+
+  // issue: https://github.com/simple-icons/simple-icons-website/issues/66
+  it('does not have a black-like color at the start', async () => {
+    const colors = [
+      '181717',
+      '1D1717',
+      '000000',
+      '512BD4',
+      '40AEF0',
+      '0094F5',
+      'FF0000',
+      '004088',
+      '0099E5',
+      'EF2D5E',
+      'FF9E0F',
+      '071D49',
+      '00A98F',
+      '191A1B',
+      '41454A',
+      'A100FF',
+      '26689A',
+      'A9225C',
+      '83B81A',
+      '0085CA',
+      '0B2C4A',
+    ];
+
+    const hexes = sortByColors(colors);
+    console.log(hexes);
+    // should not match black colors
+    await expect(hexes[0]).not.toMatch('181717');
+    await expect(hexes[1]).not.toMatch('1D1717');
```

Comment:
Why not test if output of the sorting algorithm against an expected list of hexes, rather than just checking if the first and second are not some particular elements in the list?

At that point, the comment referring to #66 also becomes obsolete as having a test like this just makes sense I would say. But you may keep it around if you think that reference is useful.

Reply:
I will keep the reference around, and also change to check the full output.